test/test/pmu/Dictionary.java

```diff
@@ -30,18 +30,18 @@ private static void testRandomRead(long[] array, int bound) {
         System.out.printf("Time spent: %.3f\n", (endTime - startTime) / 1e9);
     }
 
-    public static void test128K() {
-        long[] array = new long[1024 * 1024];
+    public static void test16K() {
+        long[] array = new long[8 * 1024 * 1024];
         testRandomRead(array, 16384);
     }
 
     public static void test8M() {
-        long[] array = new long[1024 * 1024];
-        testRandomRead(array, 1024 * 1024);
+        long[] array = new long[8 * 1024 * 1024];
```

Comment:
8M meant 8 Megabytes, you made it 64. Perhaps, we can rename it to `test8MB` to avoid confusion.
Does the test fail with 8MB?

Reply:
Tests fail with 8MB: 

```
WARNING: PmuTests.cacheMisses failed
java.lang.AssertionError: Expected 0.6086956521739131 < 0.2
        at one.profiler.test.Assert.isLess(Assert.java:18)
        at test.pmu.PmuTests.cacheMisses(PmuTests.java:40)
```

Reply:
I can rename the functions to test16KB/test8MB or testSmallRead/testLargeRead.